cai_causal_graph/graph_components.py

```diff
@@ -666,6 +584,66 @@ def to_dict(self, include_meta: bool = True) -> dict:
         }
 
         if include_meta:
-            edge_dict['meta'] = self.meta
+            edge_dict['meta'] = self.meta.copy()
 
         return edge_dict
+
+
+class TimeSeriesEdge(Edge):
+    """
+    Class defining a time-series edge.
+
+    Time series edge is equivalent to a `cai_causal_graph.graph_components.Edge` class, except that it only
```

Comment:
**A** time-series edge...

Would add a hyphen to be consistent with format you have used elsewhere in the docstring